tokio/src/runtime/scheduler/multi_thread/worker.rs

```diff
@@ -830,7 +839,16 @@ impl Handle {
         }
     }
 
-    fn notify_parked(&self) {
+    fn notify_parked_local(&self) {
```

Comment:
Another reason is split with `local` is the notification logic will change in the future between notifying locally and notifying from outside of the runtime.